service/terraform-plugin/src/test/java/com/solace/maas/ep/event/management/agent/plugin/terraform/TerraformUtilsTest.java

```diff
@@ -0,0 +1,38 @@
+package com.solace.maas.ep.event.management.agent.plugin.terraform;
+
+import com.solace.maas.ep.event.management.agent.plugin.command.model.CommandRequest;
+import com.solace.maas.ep.event.management.agent.plugin.terraform.manager.TerraformUtils;
+import org.junit.jupiter.api.Test;
+import org.springframework.test.context.ActiveProfiles;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+@ActiveProfiles("TEST")
+public class TerraformUtilsTest {
+
+    @Test
+    public void testDeleteConfigPath() throws IOException {
+        CommandRequest commandRequest = CommandRequest
```

Comment:
Instead of managing temporary directory lifecycle yourself, you SHOULD simply use `@TempDir` support from JUnit 

```shell
    public void testDeleteConfigPath(@TempDir tempDirectory ) throws IOException {
```

This is safer and much more concise

Reply:
https://www.baeldung.com/junit-5-temporary-directory